loritta-discord/src/main/java/com/mrpowergamerbr/loritta/commands/vanilla/social/DivorceCommand.kt

```diff
@@ -55,6 +60,19 @@ class DivorceCommand : AbstractCommand("divorce", listOf("divorciar"), CommandCa
                                     locale["$LOCALE_PREFIX.divorced", Emotes.LORI_HUG]
                             )
 					)
+					
+					try {
+						val userDM = user?.openPrivateChannel()?.await()
```

Comment:
Try using this and then remove the null safety characters:

```suggestion
						val userPrivateChannel = user?.openPrivateChannel()?.await() ?: return
```